frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx

```diff
@@ -61,6 +63,12 @@ import {
 } from "./ObjectDetail.styled";
 
 const mapStateToProps = (state: State, { data }: ObjectDetailProps) => {
+  const isLoggedIn = !!getUser(state);
+
+  if (!isLoggedIn) {
+    return {};
+  }
+
   const table = getTableMetadata(state);
   let zoomedRowID = getZoomedObjectId(state);
```

Comment:
Oh, so the issue is that it is connected to our store and depends on its shape 🤔 Here particularly it depends on the QueryBuilder state even though it can be rendered outside of it. Ideally, it would be great to untie the ObjectDetails from it but I assume it is a significant effort